### PR TITLE
Search service new layout

### PR DESCRIPTION
**Pull request by jarekzet**
First mods of Search Service Layout

Important - top bar and footer mods need to be marged in Commons project.

### CODE REVIEW

File: ui/apps/ui/src/app/collections/data/all/filters.data.ts [hunk 1]
```diff
@@ -28,6 +22,12 @@ export const allCollectionsFilters: IFiltersConfig = {
       label: 'Scientific discipline',
       type: 'multiselect',
     },
+    {
+      id: 'unified_categories',
+      filter: 'unified_categories',
+      label: 'Research step',
+      type: 'multiselect',
+    },
```
Comment (danielkryska) [inline ui/apps/ui/src/app/collections/data/all/filters.data.ts:30]: Why have you moved the filter?

**Review by danielkryska — APPROVED** [review]

File: ui/apps/ui/src/app/collections/data/all/filters.data.ts (hunk 1 above)
Comment (jarekzet) [reply ui/apps/ui/src/app/collections/data/all/filters.data.ts:30]: just wanted to change order of filters according to project:

https://www.figma.com/file/D2vgx66m1ADdBY0A46NdZ6/Search-Service?node-id=171%3A2204&t=ew28qVM7iJdnPjyX-1

**Comment by jarekzet** [comment]
There is a request for development mods - in section below:
image
we need to display "access type" and "Type" informations. Now they are a static text in template.

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

Make serach bar biger
uropean Galaxy Service is tagged as EOSC::Galaxy Workflow SC::Galaxy WorkflowEOSC::Galaxy Workflow

**Review by agpul — DISMISSED** [review]
image (7)
 Improve the search bar

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
Please change style for selected filter
image

**Comment by jarekzet** [comment]
> image (7)
> 
> Improve the search bar

added fix for that

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV

image

**Review by danielkryska — APPROVED** [review]

**Review by Marcelinna — APPROVED** [review]

**Pull request merged**